Framework/Reflectometry/test/LoadILLReflectometryTest.h

```diff
@@ -14,15 +14,15 @@
 #include "MantidAPI/FrameworkManager.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/SpectrumInfo.h"
-#include "MantidDataHandling/LoadILLReflectometry.h"
 #include "MantidDataObjects/TableWorkspace.h"
 #include "MantidGeometry/Instrument.h"
 #include "MantidGeometry/Instrument/RectangularDetector.h"
 #include "MantidKernel/Unit.h"
+#include "MantidReflectometry/LoadILLReflectometry.h"
```

Comment:
I'm not sure if the loaders shall also be moved? Should they not stay in DataHandling?

Reply:
I agree. I'll move them back :+1: